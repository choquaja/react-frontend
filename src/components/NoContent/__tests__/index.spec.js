import React from 'react';
import { shallow } from 'enzyme';
import NoContent from '../';

describe('NoContent', () => {
  it('renders without crashing', () => {
    shallow(<NoContent />);
  });

  it('renders with a child ', () => {
    const component = shallow(<NoContent><div>Test</div></NoContent>);
    expect(component).toMatchSnapshot();
  });
});
