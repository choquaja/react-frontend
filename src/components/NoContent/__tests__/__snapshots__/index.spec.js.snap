// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoContent renders with a child  1`] = `
<NoContent__NoContentContainer>
  <NoContent__NoContentText>
    <div>
      Test
    </div>
  </NoContent__NoContentText>
</NoContent__NoContentContainer>
`;
